x-pack/test/functional/apps/lens/show_underlying_data.ts

```diff
@@ -17,6 +17,10 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const browser = getService('browser');
 
   describe('show underlying data', () => {
+    before(async () => {
+      await browser.setLocalStorageItem('discover:docExplorerUpdateCalloutClosed', 'true');
```

Comment:
@Dmitriynj   this test is using multiple windows, Lens is opening a Discover window.
documentation of `browser.setLocalStorageItem` says

"sets a value in local storage for the focused window/frame."

I think that's why it doesn't work in this case